lambda-functions/jira-bot-filter-task-runner/condition-checker.js

```diff
@@ -0,0 +1,51 @@
+const moment = require('moment');
+const config = require('./config');
+
+lastHumanActivityDate = issue => {
+  const humanHistories = issue.changelog.histories.filter(history => {
```

Comment:
this was intentional. even if the author comments or does something to the ticket, we still want to count it as an activity and reset the counter. 

if you think otherwise, i can make changes.